js_modules/dagster-ui/packages/ui-core/src/launchpad/LaunchpadSession.tsx

```diff
@@ -519,8 +519,8 @@ const LaunchpadSession: React.FC<LaunchpadSessionProps> = (props) => {
       } else {
         runConfigYaml = yaml.stringify(
           merge(
-            yaml.parse(sanitizeConfigYamlString(partition.runConfigOrError.yaml)),
             yaml.parse(sanitizeConfigYamlString(currentSession.runConfigYaml)),
+            yaml.parse(sanitizeConfigYamlString(partition.runConfigOrError.yaml)),
```

Comment:
This is the fix, had to apply the partition config onto the run config 😰

Reply:
Oh interesting, it was an order of operations thing? 🤯 Good catch!